modules/network/src/main/java/org/apache/ignite/internal/network/message/InvokeRequest.java

```diff
@@ -34,4 +34,9 @@ public interface InvokeRequest extends NetworkMessage {
 
     /** Request's message. */
     NetworkMessage message();
+
+    @Override
+    default String toStringForLightLogging() {
+        return getClass() + ": [correlationId=" + correlationId() + ", message=" + message().getClass() + ']';
```

Comment:
Not sure that I get it. Why do we need both `InvokeRequest.getClass()` and `message.getClass()`?

Reply:
The message itself, which would be interesting, is nested, for now for convenience.